packages/shared/src/mock/metadata.ts

```diff
@@ -151,10 +151,10 @@ export const mockMetadata = makeNxGroup(mockFilepath, 'NXroot', {
         }),
       ],
     }),
-    makeNxGroup('nexus_entry_no_default', 'NXentry', {
+    makeNxGroup('nexus_no_default', 'NXprocess', {
```

Comment:
Replacing the class here to check that we no longer care about it.

Reply:
> to check that we no longer care about it.

Rather to test that `NXprocess` is treated as `NXentry`.

Reply:
Hmm, for the NX badge yes, but not for the implicit child resolution -- if the `NX_class` is not `NXdata`, then we don't care what it is; it's the `NX_class` of the children that we care about. Does that make sense?